module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FeedbackTest.php

```diff
@@ -160,4 +160,43 @@ public function testFeedbackFormWithCaptcha()
             $this->findCss($page, '#modal .alert-success')->getText()
         );
     }
+
+    /**
+     * Test feedback form with the interval captcha.
+     *
+     * @return void
+     */
+    public function testIntervalCaptcha()
+    {
+        $page = $this->setupPage(
+            [
+                'Captcha' => [
+                    'types' => ['interval'],
+                    'forms' => 'feedback',
+                    'action_interval' => 8
```

Comment:
How did you come upon this number? Just wondering if we can get away with less, since adding 8 seconds to the test suite isn't ideal.

Reply:
Perhaps a simple unit test would suffice... and in any case, a unit test would be a nice complement to this. (I don't mind writing one if that would help).

Reply:
I updated the test to avoid the delay. What do you think? That doesn't exclude adding a unit test. I'll see what I can do.

Reply:
Unit tests added. Please check whether they align with your expectations!